### DISCUSSION TITLE
v3.0.0

### DISCUSSION THREAD

**Discussion by donavanbecker** [donavanbecker/homebridge-noip#148] [Announcements]
### What's Changes
- Moved from CommonJS to ES Module
- Housekeeping and updated dependencies.

**Full Changelog**: https://github.com/donavanbecker/homebridge-noip/compare/v2.0.4...v3.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/donavanbecker/homebridge-noip/releases/tag/v3.0.0'>v3.0.0</a>.</em>

**Comment by drpimples** [comment]
Version 3 config screen is blank and frozen when I try to configure it.  2.0.4 works fine.  Once I update to 3.0.0 it is a blank config screen again.

**Reply by donavanbecker** [reply]
please open issue and provide info.

Also try beta: v3.0.1-beta.1
